RELEASE.md

```diff
@@ -8,17 +8,22 @@
 
 ## Migration guide from Kedro 0.18.* to 0.19.*
 
-# Upcoming Release 0.18.10
+# Upcoming Release 0.18.11
 
 ## Major features and improvements
-* Added support for variable interpolation in the catalog with the `OmegaConfigLoader`.
 
 ## Bug fixes and other changes
 
 ## Breaking changes to the API
 
 ## Upcoming deprecations for Kedro 0.19.0
 
+# Release 0.18.10
+
+## Major features and improvements
+* Rebrand across all documentation and Kedro assets.
```

Comment:
@stichbury @tynandebold @yetudada is this wording okay?

Reply:
Maybe remove the word `across`? Fine with me either way though!